playbooks/common/openshift-cluster/upgrades/upgrade_nodes.yml

```diff
@@ -12,6 +12,13 @@
       name: lib_openshift
     static: yes
 
+  # Run the pre-upgrade hook if defined:
+  - debug: msg="Running node pre-upgrade hook {{ openshift_node_upgrade_pre_hook }}"
+    when: openshift_node_upgrade_pre_hook is defined
+
+  - include_tasks: "{{ openshift_node_upgrade_pre_hook }}"
```

Comment:
`include_tasks` is not allowed to be used in Ansible 2.3 (which is required for 3.7). `include` should be used instead

Reply:
hmm make sense